cpp/include/cudf/hashing.hpp

```diff
@@ -35,8 +35,14 @@ namespace cudf {
  * @returns A column where each row is the hash of a column from the input
  */
 std::unique_ptr<column> hash(table_view const& input,
+                             hash_id hash_function                     = hash_id::HASH_MURMUR3,
                              std::vector<uint32_t> const& initial_hash = {},
                              rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+std::unique_ptr<column> murmur_hash3_32(
```

Comment:
We don't want the same function exposed two ways. Either the Python should be updated to call the `hash` with the `HASH_MURMUR3` enum or get rid of the `hash` API and add a specific MD5 hash. I'd opt for the former.

Reply:
I mirrored the `hash_id` enum over to the python side and removed the murmur hash specific function.